web-registry/src/hooks/useQueryClient.ts

```diff
@@ -0,0 +1,104 @@
+import { useState, useEffect } from 'react';
+
+import { QueryClientImpl as BankQueryClient } from '@regen-network/api/lib/generated/cosmos/bank/v1beta1/query';
+import { QueryClientImpl as EcocreditQueryClient } from '@regen-network/api/lib/generated/regen/ecocredit/v1alpha1/query';
+import { QueryClientImpl as BasketQueryClient } from '@regen-network/api/lib/generated/regen/ecocredit/basket/v1/query';
+
+import { useLedger } from '../ledger';
+
+// TODO: map clients and ledger queries here, hidding this details from business hooks
+// TODO: Map the clients and queries here, so that the implementation details
+// of the queries are transparent to business hooks. Instead, the hook would be called
+// like this:
+// ie. useQueryClient('basket', { basketDenom })
+// ie. useQueryClient('denomMetadata', { denom: basketDenom })
```

Comment:
as discussed on slack, this wouldn't work: https://regen-network.slack.com/archives/C01LX9E8QN8/p1649866623584679?thread_ts=1649865324.300319&cid=C01LX9E8QN8 so remove or update this comment